pom.xml

```diff
@@ -4,17 +4,43 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.5.0</version> <!-- {x-version-update;org.springframework.boot:spring-boot-starter-parent;external_dependency} -->
+  </parent>
+
   <groupId>com.azure.spring</groupId>
   <artifactId>azure-spring-boot-samples</artifactId>
   <packaging>pom</packaging>
-  <version>1.0.0</version><!-- Need not change for every release-->
+  <version>1.0.0</version>
 
 
   <properties>
-    <maven.compiler.source>11</maven.compiler.source>
-    <maven.compiler.target>11</maven.compiler.target>
+    <azure-spring-cloud.version>2.9.0</azure-spring-cloud.version>
+    <azure-spring-boot.version>3.9.0</azure-spring-boot.version>
```

Comment:
I prefer not using properties if they only used for one time. Write the version in `<version></version>` is more straight forward.